feature/dashboard/src/main/java/com/simprints/feature/dashboard/settings/syncinfo/usecase/ObserveSyncInfoUseCase.kt

```diff
@@ -210,6 +210,7 @@ internal class ObserveSyncInfoUseCase @Inject constructor(
 
         val recordsTotal = when {
             isEventSyncInProgress -> null
+            projectId.isBlank() -> null // without project ID, repository access attempts will throw an exception
```

Comment:
Can it also be null in some cases?

Reply:
`authStore.signedInProjectId` returns a non-null string.